suse_openstack_cloud

```diff
@@ -134,6 +136,8 @@ _capture_filter() {
         # "bmc_password": "s3kr1t",
         # "machine-install": { "password": "s3kr1t" },
         -e 's/\("\([a-z_]\+\)\?\(passw\(or\)\?d\|token\)":\? \+"\+\)[^"]\+"/\1'"$CENSORED"'"/gi'
+        # 'password': 's3kr1t',
+        -e "s/\('\([a-z_]\+\)\?\(passw\(or\)\?d\|token\)':\? \+'\+\)[^']\+'/\1$CENSORED'/gi"
```

Comment:
I guess so. I really just took the other regexp and changed the quotes (as changing the other regexp to deal with both style of quotes was making it really unreadable)